ui/components-react/src/components-react/tree/controlled/TreeEvents.ts

```diff
@@ -26,42 +26,48 @@ import type { Observable, Subscription } from "./Observable";
  */
 export interface TreeEvents {
   /** Called when tree node is expanded. */
+  // eslint-disable-next-line deprecation/deprecation
   onNodeExpanded?(event: TreeNodeEventArgs): void;
   /** Called when tree node is collapsed. */
+  // eslint-disable-next-line deprecation/deprecation
   onNodeCollapsed?(event: TreeNodeEventArgs): void;
   /** Called when selected tree node is clicked. */
+  // eslint-disable-next-line deprecation/deprecation
   onDelayedNodeClick?(event: TreeNodeEventArgs): void;
   /** Called when a tree node is double-clicked */
+  // eslint-disable-next-line deprecation/deprecation
   onNodeDoubleClick?(event: TreeNodeEventArgs): void;
   /** Called when selected tree node editor is activated. */
+  // eslint-disable-next-line deprecation/deprecation
   onNodeEditorActivated?(event: TreeNodeEventArgs): void;
   /**
    * Called when tree selection is modified.
    * If Subscription is returned it can be used to stop event handling by calling `unsubscribe()`.
    */
   onSelectionModified?(
-    event: TreeSelectionModificationEventArgs
+    event: TreeSelectionModificationEventArgs // eslint-disable-line deprecation/deprecation
   ): Subscription | undefined;
   /**
    * Called when tree selection is replaced.
    * If Subscription is returned it can be used to stop event handling by calling `unsubscribe()`.
    */
   onSelectionReplaced?(
-    event: TreeSelectionReplacementEventArgs
+    event: TreeSelectionReplacementEventArgs // eslint-disable-line deprecation/deprecation
   ): Subscription | undefined;
 
   /**
    * Called when checkbox states for nodes are changed.
    * If Subscription is returned it can be used to stop event handling by calling `unsubscribe()`.
    */
   onCheckboxStateChanged?(
-    event: TreeCheckboxStateChangeEventArgs
+    event: TreeCheckboxStateChangeEventArgs // eslint-disable-line deprecation/deprecation
   ): Subscription | undefined;
 }
 
 /**
  * Data structure that describes tree node event payload.
  * @public
+ * @deprecated in 4.13.x. Event args are inferred from a listener. If explicit type is needed use a type helper.
```

Comment:
I don't think these `Tree*EventArgs` types should be deprecated. They are not used with `BeEvent` or BeUIEvent` to describe listeners args. They are used as parameter types for functions of `TreeEvents`.

Reply:
Reverted